ibmcloud/image/Makefile

```
@@ -7,8 +7,8 @@ include ../../podvm/Makefile.inc
 
 .PHONY: build push verify ubuntu clean
 
-UBUNTU_RELEASE     = focal
-UBUNTU_IMAGE_URL  := https://cloud-images.ubuntu.com/$(UBUNTU_RELEASE)/current/$(UBUNTU_RELEASE)-server-cloudimg-$(DEB_ARCH).img
+UBUNTU_RELEASE     = jammy
+UBUNTU_IMAGE_URL  := https://cloud-images.ubuntu.com/$(UBUNTU_RELEASE)/release/$(UBUNTU_RELEASE)-server-cloudimg-$(DEB_ARCH).img
```

This might be the right link:
```suggestion
UBUNTU_IMAGE_URL  := https://cloud-images.ubuntu.com/releases/$(UBUNTU_RELEASE)/release/$(UBUNTU_RELEASE)-server-cloudimg-$(DEB_ARCH).img
```

---

Maybe we should decide on which image to use. This URL is referring to a daily Ubuntu build, where we pin a release in most other places. We should either use the the latest release everywhere or pin the release. So either

https://cloud-images.ubuntu.com/releases/jammy/release-20230719/

or

https://cloud-images.ubuntu.com/releases/jammy/release/

---

I think we pin the release so that it's reproducible

---

Updated.